### ISSUE TITLE
[Feature request] "About" section in user profile

### ISSUE THREAD

**Issue by devcline** [vector-im/riot-web#7704]
What about some about information in profiles? Only text for security reasons.

**Comment by lampholder** [comment]
Loosely related: https://github.com/vector-im/riot-web/issues/1401

Extensible profile information is (I believe) something that still needs to be implemented server side.

**Comment by WebFreak001** [comment]
here is a suggestion how it could look in the UI, so it's obvious what purpose the feature would serve and why it makes sense to implement it:

![UI mockup](https://user-images.githubusercontent.com/2035977/54808475-9b81a300-4c80-11e9-93a0-596ff2510249.png)

(more details in that referenced issue above this comment)

**Comment by Biep** [comment]
It should be hierarchical: global info, and then per community/room/thread.  In a programming discussion other info is relevant than in a sports discussion.

**Comment by turt2live** [comment]
Closing in favour of https://github.com/vector-im/riot-web/issues/1401 - Extensible profiles need to land first before we can track all these adhoc requests.
